src/watcher.js

```diff
@@ -135,6 +137,9 @@ class Watcher extends SafeEventEmitter {
                 case routes.DIRECTORY_FOLLOWING:
                     this.waitForLoad('following').then(() => this.emit('load.directory.following'));
                     break;
+                case routes.DIRECTORY_FOLLOWING_LIVE:
+                    this.waitForLoad('following').then(() => this.emit('load.directory.following'));
```

Comment:
this can cause an infinite loop.. this calls a function that clicks a link to load this ...

Reply:
Added followingLive to loadPredicates. Was going to stick with just following but in the debug it would show following when loading following/live. Wasn't sure if this was needed though since it's the exact same as following just renamed.

Added load.directory.following.live for DIRECTORY_FOLLOWING_LIVE route.